src/pages/Settings/index.js

```diff
@@ -36,6 +36,50 @@ const Advanced = loadable({
     ),
 });
 
+const DefaultInterProSettings = () => {
+  const [shouldRedirect, setShouldRedirect] = useState(
+    window.localStorage.getItem('redirect_to_interpro7'),
+  );
+  const handleChange = () => {
+    if (shouldRedirect === 'default') {
+      setShouldRedirect(null);
+      window.localStorage.removeItem('redirect_to_interpro7');
+    } else {
+      setShouldRedirect('default');
+      window.localStorage.setItem('redirect_to_interpro7', 'default');
+    }
+  };
+  return (
+    <>
+      <h4>Default InterPro Website</h4>
+      <div className={f('row')}>
+        <div className={f('medium-12', 'column')}>
+          <p>Use InterPro 7 (Beta) as your default InterPro website</p>
```

Comment:
I'd change the text to "Select this beta version as your default InterPro website". Not sure how many users will care or need to know that InterPro is version 7.